src/Codeception/Command/Shared/Config.php

```diff
@@ -47,7 +59,7 @@ protected function overrideConfig($configOptions)
         return Configuration::append($updatedConfig);
     }
 
-    protected function enableExtensions($extensions)
+    protected function enableExtensions($extensions): ?array
```

Comment:
I think that all methods in this class return `array`.
If for some reason one of Configuration properties is not initialized, the things must be so bad that execution is stopped before these methods can possibly be executed.

Reply:
Done.
> I think that all methods in this **class** return `array`.

Now that you mention it...
I would like to add the suffix 'Trait' ... to the names of those traits.
So that it looks like: `ConfigTrait` `FileSystemTrait` and `StyleTrait`.
It is a convention to include these suffixes in Traits and Interfaces, but I don't see it very often in Codeception.
Do you agree with them being renamed?